person.rb

```diff
@@ -0,0 +1,22 @@
+# Person class
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, parent_permission = 'true', name = 'unknown')
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission == true
+  end
```

Comment:
- [ ] This method shouldn't be private.  Kindly move this method above the "private" keyword so it becomes public.

Reply:
@Meltrust, Thank you so much for your review :star: